src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/index.jsx

```diff
@@ -15,12 +16,12 @@ const AnswerWidget = ({
           Answers
```

Comment:
For our note: we should make sure to i18n all strings using formatted message components instead of bare strings.

Reply:
Noted :)